pyrebase/pyrebase.py

```diff
@@ -45,6 +45,14 @@ def __init__(self, config):
                 "https://www.googleapis.com/auth/cloud-platform"
             ]
             self.credentials = ServiceAccountCredentials.from_json_keyfile_name(config["serviceAccount"], scopes)
+        elif config.get("serviceAccountJson"):
+            self.service_account_json = config["serviceAccountJson"]
```

Comment:
I added this since the other one was setting `self.service_account`, but I didn't notice it being used anywhere.